numba/np/ufunc/_internal.c

```diff
@@ -344,6 +344,29 @@ init_ufunc_dispatch(void)
     return result;
 }
 
+
+NUMBA_EXPORT_FUNC(PyObject)
```

Comment:
I'm also not too familiar with this C function declarations of Numba so I might be doing something wrong here. 
(Looking at the failing build)

Reply:
I don't think you need to wrap the return type in `NUMBA_EXPORT_FUNC` (at least for now) - can you try without it?

Reply:
Also, I think this should be returning a `PyObject*` since, that's what a `PyCFunctionWithKeywords` returns: https://docs.python.org/3/c-api/structures.html#c.PyCFunctionWithKeywords